src/data/taskList.ts

```diff
@@ -46,13 +46,25 @@ export const bizCaseStatus = (
   businessCase: BusinessCaseModel
   // eslint-disable-next-line consistent-return
 ) => {
+  // This is for our NEW and CURRENT intake statuses
+  // until all the old ones are deprecated
+  switch (intakeStatus) {
```

Comment:
Currently, not ALL of the new statuses are implemented. That means some of the statuses of the task list will be still based off of the `businessCase.status`.

This first switch statement is intended to catch all of the NEW statuses from the INTAKE.

At the end of all the status reworking, we probably don't need a function for this. We can just pass the status as an argument.